### ISSUE TITLE
Ability to change download client on multiple indexers at once.

### ISSUE THREAD

**Issue by sonjamichelle** [Prowlarr/Prowlarr#2234]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

No

### Describe the solution you'd like

The ability to specify a download client for an indexer was added in issue #1005.

At the moment you need to edit each indexer individually to change the preferred download client. If you have many indexers, say 40+, this can be a daunting task.

The current Select Indexers and Click edit dialog allows mass changes, but Download Client is not one of them.

![image](https://github.com/user-attachments/assets/6947c097-36b0-4ed9-bec1-27ced5567da3)

### Describe alternatives you've considered

One by One editing.

### Anything else?

No

**Comment by rumplin** [comment]
Not sure if you search from Prowlarr directly, but there is this text in the app: _"If you intend to do searches directly within Prowlarr, you need to add Download Clients. Otherwise, you do not need to add them here. For searches from your Apps, the download clients configured there are used instead.
Download clients are for Prowlarr in-app searches only and do not sync to apps. There are no plans to add any such functionality."_

**Comment by sonjamichelle** [comment]
Yes, I do, search from time to time directly from Prowrlarr when I am searching for something that another arr isn't looking for.

I found that when I was looking for something, it added the found item to my regular VPN secured download client AND to my "trusted" non secured download client. Of course, my ISP sniffed it, and I got a "warning"

Every so often I search for something that is handled by my "trusted" arr because it is not available on that particular site/tracker and I want to see if it is available elsewhere. If I find it elsewhere, I then add it to the appropriate client manually. 

Yes, a strange and quite possibly and unique way of doing things, but it works for when I'm hunting for something that is odd, new, or very old.
